platform/android/java/lib/src/org/godotengine/godot/GodotActivity.kt

```diff
@@ -93,22 +94,26 @@ abstract class GodotActivity : FragmentActivity(), GodotHost {
 	}
 
 	private fun terminateGodotInstance(instance: Godot) {
-		if (godotFragment != null && instance === godotFragment!!.godot) {
-			Log.v(TAG, "Force quitting Godot instance")
-			ProcessPhoenix.forceQuit(this)
+		godotFragment?.let {
+			if (instance == it.godot) {
```

Comment:
Here the original logic has a triple equal `===` for referential equality; looks like it was inadvertently converted to a double equals `==`.

Reply:
Ah, good catch, I missed that, thanks! Will fix...

Reply:
Fixed